src/Users.js

```diff
@@ -0,0 +1,15 @@
+import react, { useSyncExternalStore } from "react";
+import ReactDOM from "react-dom";
+
+const name ='Manisha Sinha';
+
+const element =(
+    <div className="person-item">
+        <p>{name}</p>
+
+    </div>
+);
+
+ReactDOM.render(element,document.getElementById('root'));
+
+export default Users;
```

Comment:
You are exporting a variable called "Users" but no such variable exists in this file 🤔

You are also not importing anything from `Users.js` in the rest of the application.

What are you trying to do?